orcid-api-common/src/main/java/org/orcid/api/common/writer/rdf/RDFMessageBodyWriter.java

```diff
@@ -427,6 +453,20 @@ private boolean isWebID(String urlName) {
         return urlName.equals(URL_NAME_WEBID);
     }
 
+    private boolean isInbox(String urlName) {
+        if (urlName == null) {
```

Comment:
This code followed existing code pattern. Can we make a separate PR for this change since it applies to all the checks in `describeResearcherUrls`?